README.md

```diff
@@ -486,7 +486,7 @@ Whether to skip the repository checkout phase. This is useful for steps that use
 
 **Important**: as the code repository will not be available in the step, you need to ensure that the docker compose file(s) are present in some way (like using artifacts)
 
-### `skip-pull` (optional, run only)
+### `skip-pull` (optional)
```

Comment:
It should be build and run only (as it wouldn't apply to the `push`)

Reply:
Ah thanks, fixed in latest!